data_model/src/query.rs

```diff
@@ -1135,6 +1135,48 @@ pub mod asset {
         pub account_id: EvaluatesTo<AccountId>,
     }
 
+    /// [`FindAssetRegisteringTransaction`] Iroha Query gets [`AssetId`] as input and finds
+    /// [`Hash`] of the transaction that registered this asset.
+    #[derive(
+        Debug,
+        Display,
+        Clone,
+        PartialEq,
+        Eq,
+        Decode,
+        Encode,
+        Deserialize,
+        Serialize,
+        IntoSchema,
+        PartialOrd,
+        Ord,
+        Hash,
+    )]
+    #[display(fmt = "Find asset registering transaction of `{id}` asset")]
+    pub struct FindAssetRegisteringTransaction {
+        /// `Id` of an [`Asset`] .
+        pub id: EvaluatesTo<AssetId>,
+        /// `Id` of an [`Account`] owning the asset .
+        pub account_id: EvaluatesTo<AccountId>,
+    }
```

Comment:
Why you need both `asset_id` and `account_id` if you can infer `account_id` from `asset_id`?

Reply:
Doubt we could infer for definitions.